### ISSUE TITLE
[VSCode Extension] Does not show options in a type list

### ISSUE THREAD

**Issue by javiercbk** [1pkg/gopium#21]
The vscode extension fails to detect a struct when dealing with a list of types, for example:

```go
type WillBeDetected struct{}

type (
  WillNotBeDetected struct {}
)
```

**Comment by 1pkg** [comment]
@javiercbk thank you for reporting this, I will take a look.

**Comment by 1pkg** [comment]
Indeed the problem is extension. For a broader context `gopium` extension is built on top of original vscode-go extension to utilize function like dependencies downloading via go get and etc. The problem is in original vscode-go extension code "end type" of any type is detected via this regex conditions:
```
if (decl.type === 'type') {
  const line = document.lineAt(document.positionAt(start));
  const regexStruct = new RegExp(`^\\s*type\\s+${decl.label}\\s+struct\\b`);
  const regexInterface = new RegExp(`^\\s*type\\s+${decl.label}\\s+interface\\b`);
  decl.type = regexStruct.test(line.text) ? 'struct' : regexInterface.test(line.text) ? 'interface' : 'type';
}
```

This can't be easily changed, but luckily I forked the tool goutline tool https://github.com/1pkg/goutline that originally provides type definition for the extension and in fact is the second dependency for gopium extension itself. This way I updated the tool here https://github.com/1pkg/goutline/commit/67bcd6401f0fca9008e49a8d72e559a433bc6e6f to provide more info for extension to skip this regex at all.
